### PR TITLE
Support more show statements

### PR DESCRIPTION
**Pull request by shendiaomo**

### CODE REVIEW

File: pkg/sql/executor_normal_stmt.go [hunk 1]
```diff
@@ -38,7 +47,7 @@ func isQuery(slct string) bool {
 	if strings.HasPrefix(s, "SELECT") && !has(s, "INTO") {
 		return true
 	}
-	if strings.HasPrefix(s, "SHOW") && (has(s, "CREATE") || has(s, "DATABASES") || has(s, "TABLES")) {
+	if strings.HasPrefix(s, "SHOW") && containsAnyString(s, "CREATE", "DATABASES", "TABLES", "COLUMNS", "TABLE", "PROCESSLIST") {
```
Comment (tonyyang-svail) [inline pkg/sql/executor_normal_stmt.go]: For discussion, maybe we can remove this `isQuery` check and forward all standard SQL to `db.Query`? I am not sure if `db.Exec` is useful...

Comment (shendiaomo) [reply pkg/sql/executor_normal_stmt.go]: `db.Exec` returns a https://golang.org/pkg/database/sql/#Result object, which seems cannot be acquired from a call to `db.Query`. This is the only difference I can found.
The [MySQL C++ client](https://dba.stackexchange.com/questions/24542/execute-vs-executequery-in-mysql-connector-c) recommends we call `execute` to check whether a statement is an INSERT, UPDATE, or DELETE statement. Unfortunately, the go-database module doesn't provide a similar mechanism.

File: pkg/sql/executor_normal_stmt.go [hunk 2]
```diff
@@ -31,17 +31,16 @@ func runNormalStmt(wr *pipe.Writer, slct string, db *database.DB) error {
...
 // https://golang.org/pkg/database/sql/#DB.Exec .
-// We will need to extend our parser to be a full SQL parser in the future.
-func isQuery(slct string) bool {
-	s := strings.ToUpper(strings.TrimSpace(slct))
-	has := strings.Contains
-	if strings.HasPrefix(s, "SELECT") && !has(s, "INTO") {
+// The implementation is based on
+// https://github.com/xo/usql/blob/490ef67b5accd38d53cbe0a333bef85480a0a716/drivers/qtype.go#L8-L22
+// We may need to call SQLFlow parser to check the statement type in the future.
+func isQuery(stmt string) bool {
```
Comment (wangkuiyi) [inline pkg/sql/executor_normal_stmt.go:37]: @shendiaomo `isQuery` is a hack according to the function comment. It means that we need to get rid of it ASAP. It is not a good idea to implement any new idea by extending it.

Suppose that the user writes a SQL statement like

```sql
SELECT * FROM sqlflow_history WHERE stmt="SELECT * FROM a TO TRAIN b INTO c";
```

the following condition would easily get confused:

```go
if strings.HasPrefix(s, "SELECT") && !strings.Contains(s, "INTO") {
```

**Review by wangkuiyi — COMMENTED** [review]
@shendiaomo Could you please think about how to get rid of the hack of `isQuery` from the codebase? Thanks!

File: pkg/sql/executor_normal_stmt.go (hunk 2 above)
Comment (shendiaomo) [reply pkg/sql/executor_normal_stmt.go:37]: I'll check it out.

**Pull request closed without merging**